config/common/errors.go

```diff
@@ -27,6 +27,9 @@ var (
 	ErrInvalidSourceConfig    = errors.New("source config is invalid")
 	ErrInvalidGeneratedConfig = errors.New("config generated was invalid")
 
+	// deprecated variant/version
+	ErrRhcosVariantDeprecated = errors.New("this variant is deprecated and will be removed in a future release; use openshift instead")
```

Comment:
Bikeshed, maybe:

```suggestion
	ErrRhcosVariantDeprecated = errors.New("this variant is deprecated and will be removed in a future release; use openshift variant instead")
```

just so it's clear we're not talking about some other OpenShift tool.